src/ncp/ncp_base_mtd.cpp

```diff
@@ -2067,7 +2062,7 @@ template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_CHANNEL_MONITOR_CHANN
 {
     otError error = OT_ERROR_NONE;
 
-    for (uint8_t channel = otLinkGetPhyChannelMin(mInstance); channel <= otLinkGetPhyChannelMax(mInstance); channel++)
+    for (uint8_t channel = Phy::kChannelMin; channel <= Phy::kChannelMax; channel++)
```

Comment:
Should avoid directly referencing internal symbols when possible.

Reply:
Reverted, still use `otLinkGetPhyChannelMin()` and `otLinkGetPhyChannelMax()`.